cbits/is-valid-utf8.c

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (c) 2021, Koz Ross <koz.ross@retro-freedom.nz>
+ *
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *     * Redistributions of source code must retain the above copyright
+ *       notice, this list of conditions and the following disclaimer.
+ *     * Redistributions in binary form must reproduce the above copyright
+ *       notice, this list of conditions and the following disclaimer in the
+ *       documentation and/or other materials provided with the distribution.
+ *     * Neither the name of the <organization> nor the
```

Comment:
I'd suggest either copy existing `LICENSE` verbatim, or just say `License: BSD3` and be with it.